LDAR_Sim/src/virtual_world/sites.py

```diff
@@ -154,27 +164,107 @@ def _create_equipment_groups(
                 )
         elif isinstance(equipment_groups, (int, float)) and equipment_groups == 0:
             # special case for when equipment group isn't set
-            equip_count = math.ceil(
+            if (
                 propagating_params[Infrastructure_Constants.Sites_File_Constants.REP_EMIS_EPR]
-                * 365
-                * 2
-            )
-            equip_group_info = pd.Series({PLACEHOLDER_EQUIPMENT: equip_count})
+                is None
```

Comment:
It would also be good to check that the production rates are >0 as well as not None